### PR TITLE
Fix formatDate timezone presentation

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

File: src/org/exist/xquery/value/DurationValue.java [hunk 1]
```diff
@@ -211,18 +211,36 @@ protected Duration canonicalZeroDuration() {
...
+			case HOUR:
+				r = duration.getHours();
+				break;
+
+			case MINUTE:
+				r = duration.getMinutes();
+				break;
+
+			case SIGN:
+            	return duration.getSign();
```
Comment (shabanovd) [inline src/org/exist/xquery/value/DurationValue.java]: wrong spacing here

Comment (dizzzz) [inline src/org/exist/xquery/value/DurationValue.java]: you mean:

```java
r = duration.getSign();
break;
```

Comment (shabanovd) [inline src/org/exist/xquery/value/DurationValue.java]: mean `            	return`, it should be `				return` as 3 lines above

Comment (dizzzz) [inline src/org/exist/xquery/value/DurationValue.java]: sorry I still don't understand.....
reformatting the code shows (hides?) at least there is a kind of interesting flow, which may or may not be intended. At least some comments helps to understand why things are done this way....

**Comment by adamretter** [comment]
@shabanovd @dizzzz I just rebased the formatting into a separate commit. Hopefully that helps you?

**Comment by shabanovd** [comment]
@adamretter yes, it's ok now ... waiting for travis results

**Comment by dizzzz** [comment]
@adamretter thx :-)  but my remark contained an underlying question: is the code really intended to work this way?

clearly it is :-)

**Comment by joewiz** [comment]
Thanks @adamretter for the code and @shabanovd and @dizzzz for your reviews. This will really give eXist users wonderful expressiveness in formatting dates & times.

**Comment by adamretter** [comment]
@dizzzz Ah right sorry. Yes you are right, it was as intended :-)

**Pull request merged**